### PR TITLE
Feature/SOF-7250

### PR DESCRIPTION
**Pull request by VsevolodX**
Cleaning up and streamlining dependencies installation according to new structure for the TB-made JARVIS to ESSE conversion example notebook.

### CODE REVIEW

**Comment by VsevolodX** [comment]
My changes supposed to be only in `other/materials_designer` directory. There's no other diffs when I compare to the dev.

File: other/materials_designer/config.yml [hunk 1]
```diff
@@ -17,3 +17,25 @@
...
+    - tabulate==0.9.0
+    - sqlite3
+    - sympy==1.12
+    - uncertainties==3.1.6
+    - pandas==1.5.3
+    - express-py
+    - mat3ra-esse
+    - jarvis-tools
+    - munch
+    - setuptools
```
Comment (timurbazhirov) [inline other/materials_designer/config.yml:41]: We don't really need all of these, do we?

File: other/materials_designer/Introduction.ipynb [hunk 2]
```diff
@@ -14,13 +14,12 @@
     "\n",
```
Comment (timurbazhirov) [inline other/materials_designer/Introduction.ipynb:14]: <p>We should organize this in sections:</p><p> </p><ol><li>Builders / Transformations</li><li class="ql-indent-1">1.1. Interface Creation</li><li class="ql-indent-1">1.2. Etc</li><li>Import Data</li><li class="ql-indent-1">Import materials from JARVIS</li><li class="ql-indent-1">Import materials from File in ASE-supported formats</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: other/materials_designer/import_material_from_jarvis_db_entry.ipynb [hunk 3]
```diff
@@ -0,0 +1,164 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/import_material_from_jarvis_db_entry.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    jarvis_db_entry = json.dumps(json.loads(df[(df["jid"] == "JVASP-670")].to_json(orient="records"))[0])
</pre><p>Let's isolate the JVASP identifier to a variable</p><p> </p><p>JARVIS_ID = "JVASP-670"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (timurbazhirov) [inline other/materials_designer/import_material_from_jarvis_db_entry.ipynb:1]: <p>I'd just leave the h2 title only</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**